.eslintrc.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
```

Comment:
Switching to JS from JSON seems unnecessary but I'll allow it... does this provide linting support/fixes within the config itself since it's not JSON?